### ISSUE TITLE
GTX 760 @ CN-GPU incorrect auto config (throws error)

### ISSUE THREAD

**Issue by Ryo-RU** [fireice-uk/xmr-stak#2531]
GPU: [GTX 760](https://www.techpowerup.com/gpu-specs/geforce-gtx-760.c1857)
ALGO; CN-GPU
Version: 2.10.7
NVIDIA-SMI 418.87.00    Driver Version: 418.87.00    CUDA Version: 10.1

### Auto config:
```
"gpu_threads_conf" :
[
  // gpu: GeForce GTX 760 architecture: 30
  //      memory: 1953/1998 MiB
  //      smx: 6
  { "index" : 0,
   //too high threads and blocks value:
   "threads" : 28, "blocks" : 18,
    "bfactor" : 2, "bsleep" :  0,
    "affine_to_cpu" : false, "sync_mode" : 3,
    "mem_mode" : 1,
  },
],
```
that throws error: 
```
[CUDA] Error gpu 1: </home/dima/xmr-stak2710/xmr-stak/xmrstak/backend/nvidia/nvcc_code/cuda_core.cu>:954
terminate called after throwing an instance of 'std::runtime_error'
  what():  [CUDA] Error: invalid argument
./run_sys_cnt: line 9: 21985 Aborted                 (core dumped)
```

**Comment by Ryo-RU** [comment]
manually setting it to:

```
  { "index" : 1,
    "threads" : 8, "blocks" : 48,
    "bfactor" : 2, "bsleep" :  0,
    "affine_to_cpu" : false, "sync_mode" : 3,
    "mem_mode" : 1,
  },
```
starts it working, but not sure (yet) if it is the best possible config (notice that this is index 1 gpu)
HASHRATE REPORT - NVIDIA
| ID |    10s |    60s |    15m | ID |    10s |    60s |    15m |
|  0 |  255.5 |  246.7 |   (na) |  1 |  271.9 |  270.3 |   (na) |

**Comment by psychocrypt** [comment]
should be fixed with #2538. If not please reopen the issue
